src/components/QuranReader/index.tsx

```diff
@@ -115,10 +119,40 @@ const QuranReader = ({
     }
   };
 
+  /**
+   * select all DOM node that contains data attribute `data-word-location`
+   * then see if that node is within `selection`
+   * then get the uthmani_text for those nodes
+   * then copy the uthmani_text to clipboard
+   *
+   * We need to use `uthmani_text` instead of the rendered text
+   * because sometime user is using code_v1 and code_v2 font,
+   * which uses unicode, so it can't be copied properly by user
+   */
+  const onCopy = (event) => {
```

Comment:
wrap this in a useCallback

Reply:
I moved this function into another file as you suggested yesterday in the call. So we're not redeclaring the function inside the component anymore. 👍